spmd/compiler/distribute.py

```diff
@@ -654,7 +654,7 @@ def gather_inputs_for_compilation(
     #
     # TODO(chienchin): figure out how to be compatible with optimizer state_dict
     # loading.
-    if force_compile:
+    if optimize_first_iter:
```

Comment:
I see - this param flipped in the original code from optimize_first_iter in api.py, and auto became force_compile via positional param.
So this change simply corrects it's naming to be consistent.